### PR TITLE
 Fix SubmitJobCommand error messages

### PR DESCRIPTION
**Pull request by zeineb**

### CODE REVIEW

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 4 issues

1. MAJOR [SchedulerAuthentication.java#L196](https://github.com/ow2-proactive/scheduling/blob/97d89ae90c513ef149a9d42da963759dfc6cdc00/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/authentication/SchedulerAuthentication.java#L196): Remove this unused "s3Client" local variable. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1481)
1. MAJOR [SchedulerAuthentication.java#L197](https://github.com/ow2-proactive/scheduling/blob/97d89ae90c513ef149a9d42da963759dfc6cdc00/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/authentication/SchedulerAuthentication.java#L197): Remove this unused "credentials" local variable. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1481)
1. MAJOR [SubmitJobCommand.java#L63](https://github.com/ow2-proactive/scheduling/blob/97d89ae90c513ef149a9d42da963759dfc6cdc00/rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/sched/SubmitJobCommand.java#L63): Replace this usage of System.out or System.err by a logger. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS106)
1. MAJOR [SubmitJobCommand.java#L101](https://github.com/ow2-proactive/scheduling/blob/97d89ae90c513ef149a9d42da963759dfc6cdc00/rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/sched/SubmitJobCommand.java#L101): Remove this unused method parameter "currentContext". [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1172)

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 3 issues

1. MAJOR [SubmitJobCommand.java#L63](https://github.com/ow2-proactive/scheduling/blob/f4b48ee27b697fa492e49f8bf67647bb91e51853/rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/sched/SubmitJobCommand.java#L63): Replace this usage of System.out or System.err by a logger. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS106)
1. MAJOR [SubmitJobCommand.java#L101](https://github.com/ow2-proactive/scheduling/blob/f4b48ee27b697fa492e49f8bf67647bb91e51853/rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/sched/SubmitJobCommand.java#L101): Remove this unused method parameter "currentContext". [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1172)
1. MINOR [SchedulerAuthentication.java](https://github.com/ow2-proactive/scheduling/blob/f4b48ee27b697fa492e49f8bf67647bb91e51853/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/authentication/SchedulerAuthentication.java): Replace all tab characters in this file by sequences of white-spaces. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS00105)

File: bin/proactive-server [hunk 1]
```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env bash
```
Comment (fviale) [inline bin/proactive-server]: this pull request should not contain changes in the proactive-server script. Additionally, this script is generated by the build and should not be versioned in the git repository

File: config/scheduler/settings.ini [hunk 2]
```diff
@@ -176,15 +176,15 @@ pa.scheduler.auth.policy.refreshperiod.seconds=30
 
 # The same for the GLOBAL space. The GLOBAL space is shared between each users and each jobs.
 # (concerning the syntax, see above explanations in the INPUT SPACE section)
-#pa.scheduler.dataspace.defaultglobal.url=
+pa.scheduler.dataspace.defaultglobal.url=s3://activeeon-demo-s3
```
Comment (fviale) [inline config/scheduler/settings.ini]: this pull request should not contain changes of the config/settings.ini file

**Comment by fviale** [comment]
Could you provide a description of the purpose of this pull request ?

**Comment by fviale** [comment]
Additionally, your pull request contains 23 commits ! It cannot be merged in the current state.

**Comment by fviale** [comment]
So as we discussed, it's better to clean your fork and create a new pull request, with a description of the purpose you mentioned.

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 3 issues

1. MAJOR [SubmitJobCommand.java#L63](https://github.com/ow2-proactive/scheduling/blob/36616a378b682c4b082a2778734a357ea2a977bf/rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/sched/SubmitJobCommand.java#L63): Replace this usage of System.out or System.err by a logger. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS106)
1. MAJOR [SubmitJobCommand.java#L101](https://github.com/ow2-proactive/scheduling/blob/36616a378b682c4b082a2778734a357ea2a977bf/rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/sched/SubmitJobCommand.java#L101): Remove this unused method parameter "currentContext". [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1172)
1. MAJOR [SubmitJobCommand.java#L158](https://github.com/ow2-proactive/scheduling/blob/36616a378b682c4b082a2778734a357ea2a977bf/rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/sched/SubmitJobCommand.java#L158): Move the "xml" string literal on the left side of this string comparison. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1132)

**Pull request closed without merging**